File: projet-crud/src/app/models/ReservationDTO.ts
import { UtilisateurDTO } from './UtilisateurDTO';
import { TerrainDTO } from './TerrainDTO';

export interface ReservationDTO {
  id?: number;
  utilisateur: UtilisateurDTO;
  terrains: TerrainDTO[];
  commentaire?: string;
}

File: poly-gemimi/src/main/java/fr/polytech/poly_gemimi/dto/ReservationDTO.java
package fr.polytech.poly_gemimi.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.Set;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReservationDTO {
    private Long id;
    private UtilisateurDTO utilisateur;
    private Set<TerrainDTO> terrains;
    private String commentaire;
}
